package.json

```diff
@@ -89,17 +89,15 @@
     "@cypress/vue": "^4.0.0",
     "@intlify/eslint-plugin-vue-i18n": "^2.0.0",
     "@jscutlery/semver": "^2.23.4",
-    "@nrwl/cli": "15.9.3",
```

Comment:
Is there an `nx/cli` equivalent or is it no longer needed? 🤔

Reply:
I don't think it's needed anymore. It's [replaced](https://github.com/nrwl/nx/issues/17140#issuecomment-1557239001) by `nx`  🙂